frontend/src/scenes/persons/PersonV2.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react'
+import { Row, Tabs, Col, Card, Skeleton, Tag, Dropdown, Menu, Button, Popconfirm } from 'antd'
+import { hot } from 'react-hot-loader/root'
+import { SessionsTable } from '../sessions/SessionsTable'
+import { EventsTable } from 'scenes/events'
+import { useActions, useValues } from 'kea'
+import { personsLogic } from './personsLogic'
+import { PersonHeader } from './PersonHeader'
+import './Persons.scss'
+import { CopyToClipboardInline } from 'lib/components/CopyToClipboard'
+import { midEllipsis } from 'lib/utils'
+import { DownOutlined, DeleteOutlined, MergeCellsOutlined, LoadingOutlined } from '@ant-design/icons'
+import moment from 'moment'
+import { MergePerson } from './MergePerson'
+import { PersonProperty } from './PersonProperty'
+
+const { TabPane } = Tabs
+
+export const PersonV2 = hot(_PersonV2)
+function _PersonV2(): JSX.Element {
+    const [activeTab, setActiveTab] = useState('events')
+    const [mergeModalOpen, setMergeModalOpen] = useState(false)
+
+    const { person, personLoading, deletedPersonLoading } = useValues(personsLogic)
+    const { deletePerson, setPerson } = useActions(personsLogic)
+
+    const ids = (
+        <Menu>
+            {person?.distinct_ids.map((distinct_id: string) => {
```

Comment:
Nit: This can be simplified to `distinct_id: string) => (` and drop the explicit return below